prody2/viewers/viewer_gnm.py

```diff
@@ -0,0 +1,266 @@
+# **************************************************************************
+# *
+# * Authors:     J.M. De la Rosa Trevin (jmdelarosa@cnb.csic.es)
+# *              Slavica Jonic  (slavica.jonic@upmc.fr)
+# *              James Krieger (jmkrieger@cnb.csic.es)
+# *              Ricardo Serrano Gutiérrez (rserranogut@hotmail.com)  
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+"""
+This module implement the wrappers around ProDy GNM 
+visualization programs.
+"""
+
+from pyworkflow.gui.project import ProjectWindow
+from pyworkflow.protocol.params import LabelParam, IntParam, BooleanParam
+from pyworkflow.viewer import ProtocolViewer, DESKTOP_TKINTER, WEB_DJANGO
+
+from pwem.viewers.plotter import EmPlotter
+
+from pwem.viewers import ObjectView, VmdView, DataView
+from pwem.emlib import MDL_NMA_ATOMSHIFT
+from pwem.objects import SetOfNormalModes
+
+from continuousflex.protocols import FlexProtNMA
+from continuousflex.viewers.nma_plotter import FlexNmaPlotter
```

Comment:
I'd replace this with `from pwem.viewers.plotter import EmPlotter`

That way, we don't have a dependency on continuousflex.